app/controllers/api/v1/tasks_controller.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+class Api::V1::TasksController < ApplicationController
+  before_action :authorize_request
+  before_action :task_find, only: %i[update destroy status]
+  before_action :project_find, only: %i[create]
+
+  def index
+    render json: TaskSerializer.new(@current_user.tasks).serialized_json, status: :ok
+  end
+
+  def create
+    task = @project.tasks.create(task_params)
+    if task.save
+      render json: TaskSerializer.new(task).serialized_json, status: :created
+    else
+      render json: task.errors, status: :unprocessable_entity
+    end
+  end
+
+  def update
+    if @task.update(task_params)
+      render json: TaskSerializer.new(@task).serialized_json, status: :ok
+    else
+      render json: @task.errors, status: :unprocessable_entity
+    end
+  end
+
+  def destroy
+    @task.destroy
```

Comment:
what else should be here? please, check what you did for projects controller for the same action.